tf/start.sh

```diff
@@ -42,6 +42,10 @@ do
     # train all networks
     for netarch in ${NETARCHS[@]}
     do
+      echo "Saving .yaml changes:"
+      git add $CONFIGDIR/$netarch.yaml
```

Comment:
This only works if configdir is under the git directory - which on test training is currently not true.

Reply:
This is even better. Then we can do:
```
echo "Saving .yaml changes:"
pushd $CONFIGDIR
git add $netarch.yaml
git commit -m "Configuration change"
git push https://github.com/LeelaChessZero/lczero-training-conf.git &
popd
```
and have all changes in a new "lczero-training-conf" git repository

Reply:
Sounds good to me!